### ISSUE TITLE
Downsampling of music in the web app doesn't work

### ISSUE THREAD

**Issue by KingDuckZ** [airsonic/airsonic#624]
## Problem description

In the Transcoding tab I've put the following settings:
*name:* vorbis audio
*convert from:* mp3 oga aac m4a flac wav wma aif aiff ape mpc shn
*convert to:* ogg
*step 1:* ffmpeg -i %s -map 0:0 -b:a %bk -v 0 -f ogg -acodec libvorbis -

and

*downsample command:* ffmpeg -i %s -map 0:0 -b:a %bk -v 0 -f ogg -acodec libvorbis -

but during playback in the web app these settings seem to be ignored.

### Steps to reproduce

* add some high quality song to your library (I added a 256k vbr ogg created with vorbit tools/soundkonverter)
* go to your player settings in your Airsonic web app and set the max bitrate to 128k, for example
* save, navigate back to your song and play it
* in the playlist at the bottom of the web app, note how big the file being played is (for example 10 MiB)
* you can double check the file size on your server
* now go to the Status page, under the Transmitted column see how the full 10 MiB get transmitted
* if the song was actually downsampled to 128k, I'd expect 4 or 5 MiB to be transmitted instead

## System information

 * **Airsonic version**: 10.0.1
 * **Operating system**: ArchLinux-ARM on Odroid-C2
 * **Proxy server**: tested through nginx/https only
 * **Client**: Firefox 52.4 on Gentoo Linux

## Additional notes

As discussed on IRC

**Comment by heyarne** [comment]
Are you sure this is an airsonic problem? Your config looks like it's not matching the `.ogg`:
>  *convert from:* mp3 oga aac m4a flac wav wma aif aiff ape mpc shn

**Comment by KingDuckZ** [comment]
Heyame is the conversion setting also used for downsampling then?
What I did when I changed that setting was simply swap mp3/ogg between the convert to/convert from columns. So maybe the original settings work with ogg but not mp3?
I can't test mp3 because of bug #622.

**Comment by heyarne** [comment]
To be fair: Without checking the code I don't see why conversion and downsampling shouldn't be the same thing. I always read it as "when bitrate is above X and extensions matches one of the following Ys, execute this command and stream the result".

**Comment by KingDuckZ** [comment]
I have no idea. On IRC they told me transcoding != downsampling, that's all I know. Oh and that downsampling is broken for vbr music, iirc. I'm not excluding that there could be mistakes in my config, that's why I posted it, but given what I've changed it feels like that setting would either break downsampling or transcoding. Like, say I add ogg to the "convert from" list, then would it transcode all 128k ogg to 128k ogg too? simply because it matches?
